src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,32 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class FileWork {
+    private static final char SEARCHED_LETTER = 'w';
+    private static final String SEPARATOR = " ";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> result = new ArrayList<>();
```

Comment:
Use abstraction instead implementation where it possible
```suggestion
        List<String> result = new ArrayList<>();
```